src/SearchApiManager.php

```diff
@@ -103,23 +102,6 @@ public function setOutdated(IndexInterface $index, array $ids) {
     $this->state->set('thunder_search_api_outdated_' . $index->id(), $ids);
   }
 
-  /**
-   * Check if entity is outdated.
-   *
-   * @param \Drupal\search_api\IndexInterface $index
-   *   Search API Index.
-   * @param string $id
-   *   Entity combined id.
-   *
-   * @return bool
-   *   Entity is outdated.
-   */
-  public function isOutdated(IndexInterface $index, $id) {
```

Comment:
I think it is weird that the SearchApiManager lost the ability to check for outdated ids.
Maybe there should be service, which would be injected into the FieldPlugin and the Manager class